src/logic/operate.js

```diff
@@ -8,7 +8,12 @@ const operate = (numberOne, numberTwo, operation) => {
 
   switch (operation) {
     case ('÷'):
-      result = a.div(b).toFixed(2).toString();
+			try {
+				result = a.div(b).toFixed(2).toString();
+			} catch(e) {				
+				alert("Division by zero error");
```

Comment:
You did fix the issue but the best practice is to display "error" as the result.

Don't forget it is not encouraged to use alert in production.